1-exercises/A-this/exercise-1.js

```diff
@@ -15,7 +15,10 @@ let person = {
     name: "Adrian",
     age: 32,
     currentAddress: "London",
-    celebrateBirthday: /* todo */
+    celebrateBirthday: function() {
+        newAge = age + 1;
```

Comment:
I think we were supposed to use 'this' in the code.
if you did (this.age += 1;) on this line you wont need to declare a new variable to return